tests/Integrations/Curl/CurlIntegrationTest.php

```diff
@@ -417,20 +417,22 @@ public function testAppendHostnameToServiceName()
 
     public function testHttpHeadersIsCorrectlySetAgain()
     {
-        $found = [];
+        $this->inRootSpan(function () use (&$found) {
```

Comment:
`$found` doesn't exist outside this scope, so I don't think `use (&$found)` should be there at all.

Reply:
that is a leftover from a previous configuration, very good catch 👍